<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityWidget</class>
 <widget class="QWidget" name="SecurityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>34</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SecurityWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelFieldName">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">Label:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineFieldValue"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushBrowse">
     <property name="text">
      <string notr="true">...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
